test/bigtest/tests/package-content-test.js

```diff
@@ -0,0 +1,110 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { faker } from '@bigtest/mirage';
+
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import PackageContentInteractor from '../interactors/package-content';
+
+const PKG = {
+  name: 'access_start_access_end_tests Package'
+};
+
+const testData = [
+  { pti: { titleInstance: { id: '1234', name: 'testInstance1' } }, accessStart: () => faker.date.recent().toISOString() },
+  { pti: { titleInstance: { id: '234', name: 'testInstance2' } }, accessStart: () => faker.date.future().toISOString() },
+  { pti: { titleInstance: { id: '734', name: 'testInstance3' } }, accessEnd: () => faker.date.recent().toISOString() },
+];
+
+describe('PackageContentCRUD', () => {
```

Comment:
Can you pick a better name for this? You're not really testing CRUD operations here are you?